docs/src/tutorial.md

```diff
@@ -0,0 +1,44 @@
+# Tutorial
+
+This tutorial is essentially a collection of examples.
+
+## Performance Profile
+
+Performance profiles are straightforward to use. The input is a matrix `T` with entries `T[i,j]` indicating the cost to solve problem `i` using solver `j`. Cost can be, for instance, elapsed time, or number of evaluations. The cost should be positive. Otherwise, and values will be transposed by 1.
+
+Basic usage:
+
+```@example ex1
+using BenchmarkProfiles, Plots, Random
```

Comment:
Do we need `using Plots`?

Reply:
Yes, because of the `pyplot()` below.

Reply:
Does GR not work here?

Reply:
I find PyPlot better looking but I can change.

Reply:
I find it a bit confusing because we're telling the user that it's necessary to also import `Plots` to plot a profile, which isn't true.

Reply:
I see, let me change it then

Reply:
`ylabel!` needs `using Plots`, but it can be used later.